components/stories/Checkboxes.story.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { ComponentMeta } from '@storybook/react'
+
+import { Checkboxes } from '../Checkboxes'
+
+export default {
+  component: Checkboxes,
+  title: 'Check Boxes'
+} as ComponentMeta<typeof Checkboxes>
+
+export const Default = () => (
+  <Checkboxes
+    options={['first', 'second', 'third']}
+    updateCallback={action('Option is selected')}
+  />
+)
+
+export const WithTitle = () => (
+  <>
+    <h1>How many stars are there in the sky?</h1>
+    <Checkboxes
+      options={['one', 'two', 'three', 'four', 'five']}
+      updateCallback={action('Option is selected')}
+    />
+  </>
```

Comment:
Wait is there a reason this is with an `h1`? I think it should be a `fieldset` with a `legend`

Reply:
It's just a story! I was engaging my favorite 2005 HTML best practices